test/fuzz/README.md

```diff
@@ -67,7 +67,7 @@ The fuzz test will be executed in three environments:
    //test/common/common:base64_fuzz_test`.
    
 5. Run the `*_fuzz_test` target against libFuzzer. E.g. `bazel run
-   //test/common/common:base64_fuzz_test --config asan-fuzzer`.
+   //test/common/common:base64_fuzz_test_run --config asan-fuzzer`.
```

Comment:
Could you mention that there are optional command line args provided by bazel-rules-fuzzing (link) such as timeout etc? Also mention, does this automatically use the corpus as a seed? What are the defaults?